modules/accredited_representative_portal/config/routes.rb

```diff
@@ -7,6 +7,8 @@
     post 'form21a', to: 'form21a#submit'
 
     resources :in_progress_forms, only: %i[update show destroy]
-    resources :power_of_attorney_requests, only:  %i[index show]
+    resources :power_of_attorney_requests, only: [:index, :show] do
+      resource :decision, only: [:show, :create]
```

Comment:
I'm especially curious where there is a specific customization that will give the route .../decision, but the controller name of PowerOfAttorneyRequestDecisionsController. This [guide](https://guides.rubyonrails.org/routing.html) should say so. This would be a good look.